src/main/resources/entity-types/users/simple_user_details.json5

```diff
@@ -158,6 +158,10 @@
           value: 'Text Message',
           label: 'Text Message',
         },
+        {
+          value: 'unknown',
+          label: 'Unknown',
+        },
```

Comment:
We replace null values with "unknown" for this field, but weren't providing a way for users to actually query for that (is-empty doesn't work because of our substitution, and the hard-coded list of values here makes it so there's no way to users to search for something outside the list)